### ISSUE TITLE
duplicate of exercise

### ISSUE THREAD

**Issue by Usernu11** [Asabeneh/30-Days-Of-JavaScript#385]
👋 Hello, I found a duplicate of exercise in:
👉 6th day, level 3, exercises 4 & 6

Maybe it will improve the course. 👾
Thank you! 👍

**Comment by asifurrahaman754** [comment]
yes there is a duplicate, I can correct this, please assign it to me
